lists-service/cicd/deploy/pipeline/export_config_buildspec.yaml

```diff
@@ -0,0 +1,83 @@
+version: 0.2
+###
+# This build project exports any variables needed for later stages and actions,
+# builds the template configs used to launch the CloudFormation templates
+
+env:
+  shell: bash
+  git-credential-helper: yes
+  variables:
+    DEBIAN_FRONTEND: "noninteractive"
+  exported-variables:
+    - CODEBUILD_BUILD_NUMBER
+    - BASE_STACK_FILE_PFIX
+    - PRODUCT_NAME
+    - PRODUCT_COMPONENT
+    - SLACK_DEPLOY_NOTIFICATION
+    - SLACK_ALERT_CHANNEL
+    - LISTS_SECRET_NAME
+    - LISTS_ECR_REPOSITORY_NAME
+    - DB_READ_ENDPOINT
+    - DB_WRITE_ENDPOINT
```

Comment:
It's good practice to have lists in alphabetical order, dupes become easy to spot and everything else is easier to find